src/database/migration/file/migration_version_8.ts

```diff
@@ -0,0 +1,26 @@
+import {SQLiteDatabase} from 'react-native-sqlite-storage';
+
+import ChatSchema from '../../schema/ChatSchema';
+import Migration from './migration.types';
+
+const TABLE_NAME = ChatSchema.getTableName();
+
+class MigrationVersion8 implements Migration {
+  up = async (db: SQLiteDatabase): Promise<void> => {
+    const upQuery = `ALTER TABLE ${TABLE_NAME}
+      ADD COLUMN attachment_json TEXT DEFAULT NULL;`;
+
+    console.log(`===== MIGRATING: ${TABLE_NAME.toLocaleUpperCase()} TABLE =====`);
+    await db.executeSql(upQuery);
+    console.log(`===== DONE MIGRATING: ${TABLE_NAME.toLocaleUpperCase()}  TABLE =====`);
+  };
+
+  down = async (db: SQLiteDatabase): Promise<void> => {
+    console.log(`===== DOWNGRADING: ${TABLE_NAME.toLocaleUpperCase()} TABLE =====`);
+    const downQuery = `ALTER TABLE ${TABLE_NAME} DROP COLUMN attachment_json`;
+    await db.executeSql(downQuery);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the down migration query is also correct but lacks error handling. SQLite does not support dropping a column directly with the `ALTER TABLE DROP COLUMN` command. You need to create a new table without the column, copy the data over, and then rename the new table. Please revise your down migration strategy.

```diff
-     const downQuery = `ALTER TABLE ${TABLE_NAME} DROP COLUMN attachment_json`;
-     await db.executeSql(downQuery);
+     // SQLite doesn't support dropping columns. You need to create a new table without the column, copy the data, and then rename the new table.
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->